samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -313,6 +317,9 @@ DEFINE_string(api, "async", api_message);
 /// @brief Number of infer requests in parallel
 DEFINE_uint64(nireq, 0, infer_requests_count_message);
 
+/// @brief inference schedule policy for MULTI
+DEFINE_string(sp, "", schedule_policy);
```

Comment:
```suggestion
DEFINE_string(sp, "DEVICE_PRIORITY", schedule_policy);
```
If `DEVICE_PRIORITY` is the default, than use it

Reply:
sorry for confusion here. has been updated labels and drafted this PR.